### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Make it a little easier for machine translation.

Please recommend a reviewer for #30086

### CODE REVIEW

File: src/WebSdk/README.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+The ASP.NET WebSdk repository contains the tasks, targets and packages required to build and publish ASP.NET and ASP.NET Core web apps.
+
+The WebSdk repository contains code for the following packages:
 
-The websdk repo contains code for the following packages
 1. Microsoft.NET.Sdk.Web.ProjectSystem
-2. Microsoft.NET.Sdk.Publish
-3. Microsoft.NET.Sdk.Web (Meta-package containing dependency to Microsoft.NET.Sdk.Web.ProjectSystem & Microsoft.NET.Sdk.Publish)
+1. Microsoft.NET.Sdk.Publish
+31. Microsoft.NET.Sdk.Web (Meta-package containing dependency to Microsoft.NET.Sdk.Web.ProjectSystem & Microsoft.NET.Sdk.Publish)
```
Comment (Rick-Anderson) [inline src/WebSdk/README.md:9]:
```suggestion
1. Microsoft.NET.Sdk.Web (Meta-package containing dependency to Microsoft.NET.Sdk.Web.ProjectSystem & Microsoft.NET.Sdk.Publish)
```

**Review by captainsafia — APPROVED** [review]

**Review by vijayrkn — APPROVED** [review]

**Comment by AraHaan** [comment]
Any plans to merge this soon?

**Comment by AraHaan** [comment]
@Rick-Anderson sorry to bother you but, mind assisting in contacting someone to review my pull request by chance?

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson sorry to bother you but, mind assisting in contacting someone to review my pull request by chance?

Can you supply the `#PR number` link to the PR?

**Comment by AraHaan** [comment]
#34618

**Pull request merged**